dgg-utilities.user.js

```diff
@@ -2321,6 +2321,11 @@ function injectScript() {
       let resultNukes;
       let result;
 
+      // exclude whispers sent using slash commands or when the main chatbox is inactive
+      if (text.startsWith("/w ") || text.startsWith("/whisper ") || !$('#chat-windows-select span[title="Destiny GG"].active').length){
```

Comment:
But if this ends up being the right approach, it's not really necessary to use jQuery here, I don't think it's used elsewhere in the script either.